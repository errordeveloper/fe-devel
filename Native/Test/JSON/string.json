""
"fred"
"a much longer string that won't fit into the short data"
"\"\/\\\b\f\n\t\r"
"piqu\u00E9"
"piqu√©"
