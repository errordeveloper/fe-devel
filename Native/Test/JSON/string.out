STRING 0 ''
STRING 4 'fred'
STRING 55 'a much longer st...'
STRING 8 '"/\\\b\f\n\t\r'
STRING 6 'piqué'
STRING 6 'piqué'
